ospo-book/chapters/glossary.md

```diff
@@ -0,0 +1,43 @@
+of these terms, but to clarify what we mean specifically under the context of this book.
+
+We believe that it is important to be concise and clear in our definitions, and to reference existing descriptions and resources whenever possible. 
+This will help us avoid creating a glossary that is overly complex or difficult to understand, and will ensure that our readers have access to the best 
+information and resources available.
+
+* **InnerSource:** is the use of open source software development best practices and the establishment of an open source-like culture within organizations
+for the development of its non-open-source and/or proprietary software. https://en.wikipedia.org/wiki/Inner_source
+
+* **InnerSource Principles:** InnerSource Principles are a set of guidelines that provide a framework for organizations 
+to tap into the collective knowledge and expertise of their employees, and to create a culture of collaboration and innovation. 
+Learn more about InnerSource principles: https://innersourcecommons.org/learn/
+
+* **Knowledge Sharing:** Knowledge sharing refers to the exchange of information and expertise between individuals, teams, and organizations. 
+Knowledge sharing is often a key component of open source and innerSource development practices. Learn more about knowledge sharing:https://en.wikipedia.org/wiki/Knowledge_sharing
+
+* **Open Source:** Open source refers to a type of software that allows users to freely use, modify, and distribute 
+the source code of a program that must comply with certain distribution criteria. Learn more about open source: https://opensource.org/osd/
+
+* **Open Source Culture:** Open source culture refers to a set of values and practices that promote collaboration, transparency, 
+and community-driven development. It is often associated with the open source software movement, but can be applied to other 
+areas as well, such as open science, open hardware, and open education. Learn more about open source culture: 
+https://en.wikipedia.org/wiki/Inner_source
+
+* **Open Source Project Dynamics:** Open source project dynamics refers to the complex social and technical interactions that occur within 
+open source communities. This includes things like peer review, community-driven governance, and decentralized 
+decision making. Understanding these dynamics is important for anyone looking to participate in or manage open source projects. 
+
+* **Open Works:** Open Works refers to the meaning of “Open” with respect to work products and projects that are released into the public domain 
+or under a license recognized by an entity such as, but not limited to, the Open Source Initiative (OSI), the Free Software Foundation GNU, 
+the Creative Commons (CC) or the Open Knowledge Foundation (OKF) with the intent of promoting a robust community of collaboration around 
+the work itself and other work that it may interoperate with. Learn more about Open Works: http://openworkdefinition.com/
+
+
```

Comment:
Futhermore I think it would be good to also include the term Free software in the glossary. There is sometimes some confusion about the difference between free software and open source software.